### PR TITLE
Initial draft of blog post on dynamic aws envs

### PR DESCRIPTION
**Pull request by burningion**
## Description

Blog post focused on dynamic AWS secrets with Doppler

## Related Issue(s)

Fixes #

## Release Notes

```release-note

```

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3542"></a>

### CODE REVIEW

File: static/images/blog/dynamic-aws-development-environments-with-doppler-and-gitpod/header.webp
Comment (Siddhant-K-code) [inline static/images/blog/dynamic-aws-development-environments-with-doppler-and-gitpod/header.webp:1]: @chrifro Do you have any other suggestion for the banner? as we are just duplicating the image from our old blog post. 

image

Comment (MichaelAring) [reply static/images/blog/dynamic-aws-development-environments-with-doppler-and-gitpod/header.webp:1]: Maybe include an abstraction of the AWS logo?

File: src/routes/blog/dynamic-aws-development-environments-with-doppler-and-gitpod.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+$ aws sts get-caller-identity
+```
+
+With this, a JSON object should pop out, letting you know who your UserId, Account, and Arn are. These should now match the role we created earlier.
+
+## Continuing to Improve the Developer Experience
+
+This just scratches the surface of what is possible with AWS integration in your development environment. From here, we could provision a database with example data from S3 for testing, add lambdas, or replay traffic in a reproduction of infrastructure from production. Thinking through the possibilities is really up to you.
+
+If you have a particularly clever idea you’d like to share with us, we want to hear about it! Please head over to [our community on Discord](https://gitpod.io/chat).
```
Comment (Siddhant-K-code) [inline src/routes/blog/dynamic-aws-development-environments-with-doppler-and-gitpod.md:115]: Let's just use this way to keep user away from a extra re-route for better user exp.

```suggestion
If you have a particularly clever idea you’d like to share with us, we want to hear about it! Please head over to [our community on Discord](https://www.gitpod.io/chat).
```

**Review by Siddhant-K-code — APPROVED** [review]
Looks great to me ❤️

**Pull request merged**